.kokoro/tests/run_tests.sh

```diff
@@ -13,61 +13,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eo pipefail
+set -xeo pipefail
 shopt -s globstar
-
-set -xe
 # We spin up some subprocesses. Don't kill them on hangup
 trap '' HUP
 
-echo "**** ENVIRONMENT ****"
-env
-
-export MAVEN_OPTS='-Xmx800m -Xms400m'
-
-# Temporary directory to store any output to display on error
-export ERROR_OUTPUT_DIR="$(mktemp -d)"
-trap 'rm -r "${ERROR_OUTPUT_DIR}"' EXIT
-
-# $1 - project
-# $2 - PATH
-# $3 - search string
-function TestIt() {
-  curl -s --show-error "https://${1}-${URL}/${2}" | \
-  tee -a "${ERROR_OUTPUT_DIR}/response.txt" | \
-  grep "${3}"
-  if [ "${?}" -ne 0 ]; then
-    echo "${1}/${2} ****** NOT FOUND"
-  fi
-}
-
+# Update gcloud and check version
+gcloud components update --quiet
+echo "********** GCLOUD INFO ***********"
+gcloud -v
+echo "********** MAVEN INFO  ***********"
+mvn -v
+echo "********** GRADLE INFO ***********"
+gradle -v
+
+# Setup required enviormental variables
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-acct.json
```

Comment:
You can turn echo off for this stuff

Reply:
Is there a way to do that? Or will ` &> /dev/null/` silence it?

Reply:
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

`set ` option of some kind.